0071-boring-protocol.md

```diff
@@ -0,0 +1,269 @@
+
+The objective is to provide greater growth to both networks while equally providing opportunity to the economics of Helium hardware operators, simultaneously creating privacy access to Helium network infrastructure. The Helium token model is described in the HIP’s presented previous to this proposal. 
+
+The BOP token utility is mainly a settlement between “clients” of the dVPN and “operators” of nodes which are used to privately access the internet. Apart from this primary function the token is staked by node operators to present loyalty to the network and financial incentive to act in “good faith” as they operate exit nodes. As nodes remain connected they increase their opportunity for traffic/data to be directed through their node and increase their profitability as the token is settled on a data consumption rate.
+
+Continued participation and discussion is greatly encouraged to ensure the proposal is in line with the needs of both communities.
+                                                                                                           
+## Motivation
+
+The drivers for this proposal are as follows:
+
+-   Privacy and Security for Helium “subscribers” and internet users as a whole
+-   Security and Scalability of the L1 as proposed in HIP-51
+-   Additional mechanisms for profitability to Helium Network infrastructure operators
+-   Proving a model of multi chain redundancy to further secure network economics and participant profitability
+-   Sustainability in manufacturing and distribution of hardware
+-   Increased Utility and Security for EVM mechanisms like wHNT
+
+A dVPN implementation would be an immediate and relevant re-purpose,  of the no longer needed processing power of ~1 million LoRa Hotspots sitting online, into the largest dVPN network right out of the box. Which provides the customer ability to switch client/ server modes in-app and add the dVPN service and BOP/dVPN token as another subDAO along with IoT and MOBILE. 
+
+The Boring Protocol code should be able to be added as a firmware update to run along the kernel in a container similar to the LoRa packet processing and forwarder. The existing hotspot network is ideal for supporting this type of tunneling traffic—as most hotspots have gigabit ethernet connections with private IPs and have adequate bandwidth, processing power and storage already. The RAK and MNTD are based on PI4s and should make a perfect mirror to test run in our distributed environment.
+                                                                                                           
+
+## Background
+
+For the Helium Network to grow globally in number of active devices and users it is necessary to develop economic incentives and opportunities for the infrastructure operators participating in supporting the network. Adding utility and profitability to the hardware involved now and in the future will solidify the Helium network as a major player across many types of networks. 
+
```

Comment:
explain here why the current mechanism (because this is what both HNT & IOT tokens are for) is not sufficient.